libs/shared/ui/src/lib/logo/logo.component.ts

```diff
@@ -0,0 +1,56 @@
+import { BooleanInput } from '@angular/cdk/coercion';
+import {
+  ChangeDetectionStrategy,
+  Component,
+  ElementRef,
+  Input,
+} from '@angular/core';
+
+import { TkLogoColor } from './color.model';
+
+@Component({
+  selector: 'tk-logo',
+  templateUrl: 'logo.component.svg',
+  styles: [
+    `
+      :host {
+        display: block;
+      }
+    `,
+  ],
+  changeDetection: ChangeDetectionStrategy.OnPush,
+})
+export class TkLogoComponent {
+  static ngAcceptInputType_color: TkLogoColor | undefined | null;
+  static ngAcceptInputType_showText: BooleanInput;
+
+  /** Theme color for the logo, defaults to 'onBackground' if no value is set. */
+  @Input()
+  get color(): TkLogoColor {
+    return this._color;
+  }
+  set color(color: TkLogoColor) {
```

Comment:
Have you seen the new `@Input({ transform })` usage? I believe it's meant to replace getter/setter